stories/DayPicker.js

```diff
@@ -52,12 +52,11 @@ storiesOf('DayPicker', module)
   .addWithInfo('with custom day size', () => (
     <DayPicker daySize={50} />
   ))
-  .addWithInfo('more than one month', () => (
```

Comment:
Just noticed there's  a pattern used in other stories that's *single month* and *3 months*, I'll go with that